apps/browser/qml/pages/components/PermissionCreateDialog.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.WebView.Controls 1.0
+
+Dialog {
+    property alias label: valueButton.label
+    property int capability: indexToCapability(valueButton.currentIndex)
+    property alias uri: textField.text
+    readonly property var _regExp: new RegExp("^(http|https):\/\/")
+
+    function indexToCapability(index) {
+        if (index === 0) { // index for allow context menu
+            return PermissionManager.Allow
+        } else {
+            return PermissionManager.Deny
+        }
+    }
+
+    canAccept: (textField.text !== "" && !textField.errorHighlight)
+
+    DialogHeader {
+        id: header
+
+        //: Accept text when add new permission
+        //% "Add"
+        acceptText: qsTrId("sailfish_browser-he-add")
+    }
+
+    Column {
+        width: parent.width
+        anchors.top: header.bottom
+
+        TextField {
+            id: textField
+
+            width: parent.width
+            //% "Website addres"
+            placeholderText: qsTrId("sailfish_browser-he-website-address")
+            label: {
+                if (errorHighlight) {
+                    //% "Should begin with http:// or https://"
+                    return qsTrId("sailfish_browser-he-error-website")
+                } else if (text == "") {
+                    return ""
+                } else {
+                    return placeholderText
+                }
+            }
+            onTextChanged: errorHighlight = (!_regExp.test(text) && text !== "")
+        }
+
+        ComboBox {
```

Comment:
This is a design issue, but it would be nice to include the icon for the permission type here, to reflect the design of the Permissions page:

![permission-icons](https://user-images.githubusercontent.com/1446122/102093974-6f319100-3e2a-11eb-8abe-8b833054efbd.png)